.drone.yml

```diff
@@ -0,0 +1,72 @@
+pipeline:
+
+# staging
+
+  build_stg:
+    image: alpine:latest
```

Comment:
Better use a fixed version, isn't it? Caddy image uses 3.6 - https://github.com/abiosoft/caddy-docker/blob/master/Dockerfile